auditbeat/module/file_integrity/monitor/tracing/int_aligned.go

```diff
@@ -0,0 +1,57 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+//go:build linux && !386 && !amd64 && !amd64p32
+
+// Alignment-safe integer reading and writing functions.
+
+package tracing
+
+import (
+	"errors"
+	"unsafe"
+)
+
+var errBadSize = errors.New("bad size for integer")
+
+func copyInt(dst unsafe.Pointer, src unsafe.Pointer, len uint8) error {
+	copy((*(*[maxIntSizeBytes]byte)(src))[:len], (*(*[maxIntSizeBytes]byte)(src))[:len])
```

Comment:
I think you want `dst` here :)

Reply:
Ha! nice one the original tracing package in `x-pack/auditbeat/tracing/int_unaligned.go` which this one is a direct copy has this issue that affects non-x86 architectures! :smile:

Reply:
@haesbaert I think you should open a PR about this in the main beats repo